core/blockchain.go

```diff
@@ -347,10 +347,8 @@ func NewBlockChain(
 
 	// If periodic cache journal is required, spin it up.
 	if bc.cacheConfig.TrieCleanRejournal > 0 && len(bc.cacheConfig.TrieCleanJournal) > 0 {
-		if bc.cacheConfig.TrieCleanRejournal < time.Minute {
```

Comment:
could we keep this as a var and modify it in tests?

Reply:
I think it's cleaner to avoid modifying it specifically for tests and don't think that this condition is really necessary. Generally I'm not a big fan of sanitizing configs and replacing the values rather than returning an error for an invalid config.